src/panels/lovelace/editor/hui-entities-card-row-editor.ts

```diff
@@ -53,6 +68,34 @@ export class HuiEntitiesCardRowEditor extends LitElement {
       return html``;
     }
 
+    if (this._editRowConfig) {
+      return html`
+        <div class="edit-entity-row-header">
+          <mwc-icon-button @click=${this._goBack}>
+            <ha-svg-icon .path=${mdiArrowLeft}></ha-svg-icon>
+          </mwc-icon-button>
+          <mwc-button
+            slot="secondaryAction"
+            @click=${this._toggleMode}
+            .disabled=${!this._editRowGuiModeAvailable}
+            class="gui-mode-button"
+          >
+            ${this.hass!.localize(
+              !this._cardEditorEl || this._editRowGuiMode
+                ? "ui.panel.lovelace.editor.edit_card.show_code_editor"
+                : "ui.panel.lovelace.editor.edit_card.show_visual_editor"
+            )}
+          </mwc-button>
+        </div>
```

Comment:
We could make this an element with a slot so every detail editor has the same base

Reply:
We should also give it a title so you know what you are editing probablyy